README.md

```diff
@@ -1,88 +1,126 @@
+
+Next, just run `pod install` from the command line in the top level directory of this project. Cocoapods will install the Twilio library and then set up a .xcworkspace file that you will use to run your project from now on.
 
-7) Copy and paste the access token into ViewController.swift (replacing the one you used earlier).
+**Note: If you've used [Twilio's Cocoapods Spec Repo](https://github.com/twilio/cocoapod-specs) before you may need to update it by running `pod repo update twilio` before installing dependencies.** 
 
-8) Build and run the app on a second physical device if you have one, or the iPhone simulator.
+#### Using an Emulator
 
-9) Once you have both apps running, enter an identical Room name (such as "MyRoom") into both apps, and tap "Connect" to connect to a video Room (you'll be prompted for mic and camera access on the physical device). Once you've connected from both devices, you should see video! (Since the iPhone Simulator doesn't have a camera you'll only see video in one direction if you're using that environment.)
+You can use the iOS Simulator that comes with Xcode to do your testing, but local video will not be shared since the Simulator cannot access a camera. 
 
-## What is this project?
+**Note:If you have an iOS device, you can now run apps from Xcode on your device without a paid developer account.**
+
+
+#### Prerequisites
+
+This project uses Apple's Swift programming language 3.0 for iOS, and the only supported way to develop iOS apps is on an Apple computer running OS X and Xcode. We have tested this application with the latest versions of iOS (10.0) and Xcode (8.0) at the time of this writing.
+
+
+#### About the Quickstart Project
 
 This project consists of two sample apps: one using CallKit, and one without CallKit support.
 
-The quick starts will help you get video chat integrated directly into your iOS applications using Twilio's Video SDK. These examples are for Swift developers - if your app uses Objective-C see [video-quickstart-objective-c](https://github.com/twilio/video-quickstart-objc/).
+The quickstarts will help you integrate Twilio Video directly into your iOS applications using Twilio's Video SDK. These examples are for Swift developers.
+
+**Note: if your app uses Objective-C see [video-quickstart-objective-c](https://github.com/twilio/video-quickstart-objc/).**
 
 The sample apps are written in Swift 3.0. You will need at least Xcode 8.0 in order to run the applications.
 
 Review the code to see how how to set up key classes like `TVIVideoClient`, `TVIRoom`, `TVIParticipant`, `TVILocalMedia`, and `TVICameraCapturer`. The ViewController implements the `TVIRoomDelegate`, and `TVIParticipantDelegate` protocols in order to display remote Participant video on screen.
```

Comment:
`TVIVideoClient` and `TVILocalMedia` have been removed from the SDK as of beta15.